.env.example

```
@@ -10,7 +10,7 @@ MYSQL_PASSWORD=your_password_here
 MYSQL_ROOT_PASSWORD=your_root_password_here
 
 # JWT Configuration
-JWT_SECRET=your_jwt_secret_key_here
+JWT_SECRET=93E65B3EF48B470DBD30D2F553A37C1F94749AC77A89BBE13D15E6C2790A7DE8A3B89EAD8CA4E5C5D6A38C3278AD7D8D
```

```suggestion
JWT_SECRET=your_jwt_secret_key_here
```
이거 이런식으로 노출되면 절대 안돼요.
.env.example 파일은 단순히 예시 파일이에요.
되돌려야할것같아요